backend/models/user.go

```diff
@@ -13,7 +13,7 @@ type User struct {
 	Common
 	Name       string `json:"name"`
 	Email      string `gorm:"unique" json:"email"`
-	IsAdmin    bool   `json:"isAdmin,default;False"`
+	IsAdmin    bool   `json:"isAdmin"`
```

Comment:
It would only change if its True, and you don't pass anything it would assume u want to make it false

Reply:
Also, I think we should put the default under gorm instead of JSON, this would keep readability and then deal with that issue aswell. 
![image](https://github.com/lynx-locks/lynx-web/assets/57304476/0159283a-0627-40b3-bdc4-07a18c9f935d)

Reply:
that works